node/pkg/admin/host/controller.go

```diff
@@ -0,0 +1,38 @@
+package host
+
+import (
+	"bisonai.com/orakl/node/pkg/admin/utils"
+	"bisonai.com/orakl/node/pkg/bus"
+	"github.com/gofiber/fiber/v2"
+	"github.com/rs/zerolog/log"
+)
+
+func getPeerCount(c *fiber.Ctx) error {
+	msg, err := utils.SendMessage(c, bus.LIBP2P, bus.GET_PEER_COUNT, nil)
+	if err != nil {
+		log.Error().Err(err).Str("Player", "Admin").Msg("failed to send message to libp2p helper")
+		return c.Status(fiber.StatusInternalServerError).SendString("failed to get peer count: " + err.Error())
+	}
+	resp := <-msg.Response
+	if !resp.Success {
+		log.Error().Str("Player", "Admin").Msg("failed to get peer count: " + resp.Args["error"].(string))
```

Comment:
```go
resp.Args["error"].(string)
```
can potentially cause runtime panic if Args doesn't contain the "error" key

Reply:
As far as I remember, there shouldn't be such cases, will double check

Reply:
every message bus fail response takes `bus.HandleMessageError` function taking error as parameter and the `error` key will always exists